lighthouse-core/test/gather/gatherers/start-url-test.js

```diff
@@ -28,18 +28,25 @@ const wrapSendCommand = (mockDriver, url, status = 200, fromServiceWorker = fals
     cb({response: {status, url, fromServiceWorker}});
   };
 
-  mockDriver.getAppManifest = () => {
-    return Promise.resolve({
-      data: '{"start_url": "' + url + '"}',
-      errors: [],
-      url,
-    });
-  };
-
   return mockDriver;
 };
 
 describe('Start-url gatherer', () => {
+  let baseArtifacts;
+
+  function createArtifactsWithURL(url) {
+    return {WebAppManifest: {value: {start_url: {value: url}}}};
+  }
+
+  beforeEach(() => {
+    jest.useFakeTimers();
+    baseArtifacts = {WebAppManifest: null};
+  });
+
+  afterEach(() => {
+    jest.advanceTimersByTime(5000);
```

Comment:
why is this needed *after* each test is done?

Reply:
the other tests not touched by this don't properly cleanup their timer, I'll add a todo to fix this another time?

Reply:
> the other tests not touched by this don't properly cleanup their timer, I'll add a todo to fix this another time?

yeah, sounds good. I think `advanceTimersByTime` is going to be a game changer for a lot of our older tests and it would be nice to update anyways